docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md

```diff
@@ -23,7 +23,5 @@ The process of collecting traces by Jaeger looks as follows:
 The process of querying traces from Jaeger looks as follows:
 
 1. A Kyma user accesses the Jaeger UI to look for specific traces.
-2. The user has [two options](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information:
```

Comment:
Can't we move that description to the "Access" document where the new diagram is to have a bit more description?

Reply:
as below, the current sentence is

`The user passes the request to the Jaeger Query service and retrieves the trace information. Learn more about [accessing Jaeger](../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md).`

Is that okay?